manifests/no-script-v2/default-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "NoScript",
+  "version": "11.4.44",
+  "url": "https://secure.informaction.com/download/releases/noscript-11.4.44-chrome.zip",
+  "sha512": "ad0dd1dba940a88eefb68487993f5a4734f3f3fd263df9a5fcd4854e23d528c5450a4fe12ceceb7d8fd381563e76bb49186fcf786895bd44cde4bc00da07deda",
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAplBwwQIQ/IvMcVuwUaFs6osApUf4g6eRKimaI6V+M0grGkdoiyJJ5210TVv86Bc/HOa/B7m22kbS/xvd+nGR5vjcdh9zSJ09B/n/VZJU3AghHJ4zJDdfxSDJo5ZBrZAB0WKn41k2amiMGKV4EfJlorjrv+sN31VuoTnbxWDwniD/DuS3CrBkoMaC7kHQnmpTe5mcGdlbWCXJVHhO+5DgXx4qBH0CIyHn6kPkiczMXAYgs3S2jWTBUpCQ8akVm5Sy942HzSp/kZdG/etvKoteTMCEcqInC3qOYbWQSpfNePDe/zitHhuACIslTuy0kc2bXCvdgxxpx1npRF86sQzfpwIDAQAB"
```

Comment:
we are going to re-signed the extensions with our keys, aren't we?

Reply:
Definitely, it's out public key